### PR TITLE
Add xhyve component for macOS #1267

### LINKED ISSUE #1267: Add xhyve component for macOS
Component should not support installation for now, but should be able to detect that xhyve driver is available.

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix #1267

### CODE REVIEW

**Comment by dgolovin** [comment]
test this

**Comment by jrichter1** [comment]
You know what this is missing? Integration with setup-cdk :)

File: requirements.json [hunk 1]
```diff
@@ -478,6 +484,41 @@
...
+    "sha256sum": "",
+    "version": "",
+    "revision": 112440,
+    "virusTotalReport": "",
+    "vendor": "macOS",
+    "messages": {
+      "info": {
+        "xhyveinfo": {
+          "message": "Follow these steps to install and enable xhyve virtualization",
+          "link": "https://docs.openshift.org/latest/minishift/getting-started/installing.html",
```
Comment (jrichter1) [inline requirements.json]: What is the point of this message? There is nothing relevant to xhyve in this document besides the link that is contained in the "xhyvedriverinfo" message.

Comment (sudhirverma) [reply requirements.json]: Thanks, @jrichter1  I will  remove xhyveinfo

**Comment by sudhirverma** [comment]
test this

**Pull request merged**